packages/walletconnect/test/shared/WalletClient.ts

```diff
@@ -93,7 +94,7 @@ export class WalletClient {
 
   constructor(provider: WalletConnectProvider, opts: Partial<WalletClientOpts>) {
     this.provider = provider
-    this.networkId = opts?.networkId ?? -1
+    this.networkId = opts?.networkId ?? ''
```

Comment:
Maybe we should restrict `networkId` to the three values (mainnet, testnet, devnet) only?